### PR TITLE
Bg/expertise paragraph

### PR DESCRIPTION
**Pull request by mutesialine**
## Describe your changes
created  expertise data   file  from data folder and removed duplicate  of data in expertise section

## Issue ticket number and link
id->43
link->[here](https://www.notion.so/apeunit/Fix-Bugs-in-the-Expertise-section-01150e75161d4b4c96d33093f6cc8c42)
## Tasks completed
- [x] created expertiseList file in data folder
- [x]  added content in expertiseList file
- [x] removed duplicate of data in expertise section
## Tasks not completed
None
## Screenshots (if needed)

### CODE REVIEW

File: components/sections/homepage/Expertise.jsx [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
           </div>
           <div className="space-y-2 md:space-y-3 pt-3 md:pt-0">
-            <p className="border-b border-grey-300">Product Design</p>
-            <p className="border-b border-grey-300">Strategy</p>
-            <p className="border-b border-grey-300">Product Design</p>
-            <p className="border-b border-grey-300">Product Design</p>
-            <p className="border-b border-grey-300">Product Design</p>
+            {dataSecondary.map((el) => (
+              <p className="border-b border-grey-300">{el}</p>
+            ))}
```
Comment (Ntwali-Josue) [inline components/sections/homepage/Expertise.jsx]: Give your paragraph a unique key value.

**Review by Ntwali-Josue — CHANGES_REQUESTED** [review]
Hey @mutesialine, Work on the review under the comments section. This will also fix your deployment issue you are having.

**Review by Ntwali-Josue — APPROVED** [review]
Well done @mutesialine and @rithaven 👍🏾

File: components/sections/homepage/Expertise.jsx [hunk 2]
```diff
@@ -1,24 +1,20 @@
 import Wrapper from "../../layout/Wrapper";
-
-export default function Expertise() {
+import { dataPrimary, dataSecondary } from "../../../data/expertiseList";
+export default function Expertise({}) {
```
Comment (izzett222) [inline components/sections/homepage/Expertise.jsx]: Could you remove the empty object destructuring?

**Review by izzett222 — CHANGES_REQUESTED** [review]

**Review by mhenriette — APPROVED** [review]

**Review by hirwaaldo1 — APPROVED** [review]
Well Done

**Review by izzett222 — APPROVED** [review]

**Pull request merged**